x-pack/plugins/search_indices/public/components/connection_details/connection_details.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  EuiButtonIcon,
+  EuiCopy,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle,
+  useEuiTheme,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useKibana } from '../../hooks/use_kibana';
+
+const DEFAULT_ELASTICSEARCH_URL = 'http://localhost:9200';
+
+export const ConnectionDetails: React.FC = () => {
+  const { euiTheme } = useEuiTheme();
+  const { cloud } = useKibana().services;
+
+  const elasticsearchUrl = cloud?.elasticsearchUrl ?? DEFAULT_ELASTICSEARCH_URL;
+
+  return (
+    <EuiFlexGroup gutterSize="s" alignItems="center">
```

Comment:
Will we need this same layout for copying the API key? if we need to match them then we might want to make this a more generic component that can be used for both.

Reply:
Yep agreed - Its likely going to be duplicated in the API key package but we can introduce a generic component if we introduce a third component with similar features.